### ISSUE TITLE
[STORY] 

### ISSUE THREAD

**Issue by mrusyd** [Australian-Imaging-Service/pipelines#184]
### Metadata

Epic: 
Feature: #65 
Feature Release: 2

### Description

Context ([user story map](https://miro.com/app/board/uXjVOrM3BF4=/?moveToWidget=3458764532745599961&cot=14)): 

- Persona: Study designer (PI/postdoc)
- Activit(y/ies): 
1. ["Define criteria for communicating with acquisition site on mismatch detection"](https://miro.com/app/board/uXjVOrM3BF4=/?moveToWidget=3458764532745599961&cot=14) (seems like an actual story already)
2. ["Define criteria for acquisition / participant exclusion"](https://miro.com/app/board/uXjVOrM3BF4=/?moveToWidget=3458764532446367018&cot=14) (potentially may be one of the acceptance criteria if not a separate story) 

NOTE: Stories placed on the board are not really stories. They are more questions and challenges, rather than stories:
1. ["How verbose to be on a mismatch? May vary depending on both nature of parameter and magnitude of difference"](https://miro.com/app/board/uXjVOrM3BF4=/?moveToWidget=3458764532442081949&cot=14)
2. ["Difference between severity of difference justifying communication with acquisition site, and justifying data exclusion"](https://miro.com/app/board/uXjVOrM3BF4=/?moveToWidget=3458764532745309952&cot=14)

Proposed and incomplete first draft (could be completely wrong): 
As a study designer (PI/postdoc), I want to define criteria for communicating with acquisition site when mismatch is detected, [SO THAT **VALUE**]

### Acceptance Criteria

> Acceptance criteria is used to confirm that the story was completed correctly. Criteria can be written in any format, or a "given-when-then" form may be used:

- [ ] 1. [ACCEPTANCE CRITERIA]

### Blocked by

- [ ]
